src/Collectors/Auth.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace CodeIgniter\Shield\Collectors;
+
+use CodeIgniter\Debug\Toolbar\Collectors\BaseCollector;
+
+/**
+ * Debug Toolbar Collector for Auth
+ */
+class Auth extends BaseCollector
+{
+    /**
+     * Whether this collector has data that can
+     * be displayed in the Timeline.
+     *
+     * @var bool
+     */
+    protected $hasTimeline = false;
+
+    /**
+     * Whether this collector needs to display
+     * content in a tab or not.
+     *
+     * @var bool
+     */
+    protected $hasTabContent = true;
+
+    /**
+     * Whether this collector has data that
+     * should be shown in the Vars tab.
+     *
+     * @var bool
+     */
+    protected $hasVarData = false;
+
+    /**
+     * The 'title' of this Collector.
+     * Used to name things in the toolbar HTML.
+     *
+     * @var string
+     */
+    protected $title = 'Auth';
+
+    //--------------------------------------------------------------------
+
+    /**
+     * Returns any information that should be shown next to the title.
+     */
+    public function getTitleDetails(): string
+    {
+        return get_class(service('auth'));
```

Comment:
Would the class of the actual authenticator used be more helpful?

Reply:
Nice!

Done.